qa/tasks/ceph_manager.py

```diff
@@ -241,8 +241,8 @@ def kill_osd(self, osd=None, mark_down=False, mark_out=False):
             with safe_while(sleep=15, tries=40, action="type ceph-objectstore-tool") as proceed:
                 while proceed():
                     proc = exp_remote.run(args=['type', 'ceph-objectstore-tool'], 
-                               wait=True, check_status=False, stdout=BytesIO(),
-                               stderr=BytesIO())
+                               wait=True, check_status=False, stdout=StringIO(),
```

Comment:
please explain why this change is not cherry-picked from master in the commit message.

Reply:
sorry, tried to address failure earlier; and then force pushed the updated commits after syncing with you. Seems like the previous cherry picks were not removed.